tests/test_argx.py

```diff
@@ -80,9 +80,9 @@ def test_extended_command_has_function_help() -> None:
 
     cli.extend_commands(sl)
 
-    help = cli.subparsers.choices[sl.bbb.__name__].format_help()
```

Comment:
for some reason, this line started to rise the pylint error:
```
python3 -m pylint aiven tests
************* Module tests.test_argx
tests/test_argx.py:83:4: W0622: Redefining built-in 'help' (redefined-builtin)
```

Reply:
It is fixed in #344 (https://github.com/aiven/aiven-client/pull/344/commits/e18d70f13e982a9776042926ac811b15c86d0f43)